frontend/mr-admin-flate/src/pages/tiltaksgjennomforinger/TiltaksgjennomforingInfo.tsx

```diff
@@ -33,6 +45,17 @@ export function TiltaksgjennomforingInfo() {
     return <Alert variant="warning">Fant ingen tiltaksgjennomføring</Alert>;
   }
 
+  const overordnetEnhet = hentEnhetsnavn(
+    enheter,
+    enheter?.find((e) => e.enhetNr === avtale?.navEnhet?.enhetsnummer)
+      ?.overordnetEnhet
```

Comment:
Var det ikke noen avtaler som _er_ overordnet enhet og ikke _har_

Reply:
Jo, det har du helt rett i 🤦‍♂️
Jeg fikser så det blir riktig for dem også

Reply:
La på samme logikk som for uthenting av overordnet enhet ved opprettelse av gjennomføring.